AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -2215,7 +2215,7 @@ stubhub.cz,stubhub.de,stubhub.dk,stubhub.es,stubhub.fi,stubhub.fr,stubhub.no,stu
 morizon.pl##.cookiesNotificationOuter
 marvelousdesigner.com##.cookiesUseAgree
 iar.com##.cookies__info
-disk-tools.com,daemon-tools.cc,oreopeople.com##.cookies_banner
+bitfinex.com,disk-tools.com,daemon-tools.cc,oreopeople.com##.cookies_banner
```

Comment:
better `bitfinex.com##.cookies`(add to existing `##.cookies`)

Reply:
Yeah you're right, fixed